DependencyInjection/Compiler/AddProcessorsPass.php

```diff
@@ -34,24 +34,24 @@ public function process(ContainerBuilder $container)
         foreach ($container->findTaggedServiceIds('monolog.processor') as $id => $tags) {
             foreach ($tags as $tag) {
                 if (!empty($tag['channel']) && !empty($tag['handler'])) {
-                    throw new \InvalidArgumentException(sprintf('you cannot specify both the "handler" and "channel" attributes for the "monolog.processor" tag on service "%s"', $id));
+                    throw new \InvalidArgumentException(\sprintf('you cannot specify both the "handler" and "channel" attributes for the "monolog.processor" tag on service "%s"', $id));
```

Comment:
Not sure about native function invocation

Reply:
> Not sure about native function invocation 

Can you elaborate? This change looks correct to me.

Reply:
Sorry, yes it looks correct, but I thought we weren't using it at symfony/symfony. I double checked and we do 👍

Reply:
Yes, we use it for compiler-optimized functions.